package.json

```diff
@@ -103,7 +103,8 @@
     "detect-browser": "^5.2.0",
     "protobufjs": "~6.8.8",
     "resize-observer": "^1.0.0",
-    "ua-parser-js": "^1.0.1"
+    "ua-parser-js": "^1.0.1",
+    "pako":"2.0.4"
```

Comment:
Is there any reason to restrict to only 2.0.4 instead of ^2.0.4?

Reply:
I had it as `^` at some point. during one of the builds the pako dependency got dropped. So then forgot to add that `^` to the new change. 

no reason, will add back.